File: unittest/scripts/js_dev_api_examples/scripts/setting_the_current_schema_2.js
// Assumptions: test schema exists
var mysqlx = require('mysqlx').mysqlx;

var session = mysqlx.getNodeSession({
  dataSourceFile: 'mysqlxconfig.json', app: 'myapp',
  dbUser: 'mike', dbPassword: 's3cr3t!'});

var default_schema = session.getDefaultSchema().name;
session.setCurrentSchema(default_schema);

// print the current schema name
print(session.getCurrentSchema().name);

File: unittest/scripts/py_dev_api_examples/scripts/setting_the_current_schema_2.py
# Assumptions: test schema exists
import mysqlx

session = mysqlx.getNodeSession({
  'dataSourceFile': 'mysqlxconfig.json', 'app': 'myapp',
  'dbUser': 'mike', 'dbPassword': 's3cr3t!'})

default_schema = session.getDefaultSchema().name
session.setCurrentSchema(default_schema)

# print the current schema name
print session.getCurrentSchema().name